content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
 
-### Glossary and FAQ {#glossary-and-faq}
+### शब्दकोष और ऍफ़.ए.क्यू. {#glossary-and-faq}
 
-The [glossary](/docs/glossary.html) contains an overview of the most common terms you'll see in the React documentation. There is also a FAQ section dedicated to short questions and answers about common topics, including [making AJAX requests](/docs/faq-ajax.html), [component state](/docs/faq-state.html), and [file structure](/docs/faq-structure.html).
+इस [शब्दकोष](/docs/glossary.html) में React के दस्तावेज़ों में सबसे ज़्यादा बार आने वाले शब्दों का एक ओवरव्यू है | यहाँ पर एक ऍफ़.ए.क्यू. भाग भी है जहा पर [AJAX रिक्वैस्ट बनाए](/docs/faq-ajax.html), [कॉम्पोनेन्ट state](/docs/faq-state.html), और [फाइल स्ट्रक्चर](/docs/faq-structure.html)  जैसे अन्य साधारण विषयों के बारे में छोटे प्रष्न और उत्तर हैं |
 
-## Staying Informed {#staying-informed}
+## जागृत रहें {staying-informed}
 
-The [React blog](/blog/) is the official source for the updates from the React team. Anything important, including release notes or deprecation notices, will be posted there first.
+[React ब्लॉग](/blog/) React टीम के द्वारा अपडेट्स के लिए एक आधिकारिक स्रोत है, इसमें रिलीज़ नोट्स और डेप्रिसिएशन की सूचनाए सबसे पहले आती हैं | 
 
-You can also follow the [@reactjs account](https://twitter.com/reactjs) on Twitter, but you won't miss anything essential if you only read the blog.
+आप ट्विटर पर [@reactjs का अकाउंट](https://twitter.com/reactjs) फॉलो कर सकते हैं, लेकिन अगर आप हमारा ब्लॉग पढ़ें तो आप कोई ज़रूरी चीज़ नहीं छोड़ेंगे | 
 
-Not every React release deserves its own blog post, but you can find a detailed changelog for every release [in the `CHANGELOG.md` file in the React repository](https://github.com/facebook/react/blob/master/CHANGELOG.md), as well as on the [Releases](https://github.com/facebook/react) page.
+हर React रिलीज़ अपने ब्लॉग पोस्ट के लायक नहीं है इसी लिए अगर आपको एक डिटेल्ड चेंज लॉग हर रिलीज़ के लिए चाहिए [तो वो `CHANGELOG.md` फाइल में है जो की React की रिपॉजिटरी में है](https://github.com/facebook/react/blob/master/CHANGELOG.md) या फिर आप [Releases](https://github.com/facebook/react) पेज पर भी जा सकते हैं |
 
-## Versioned Documentation {#versioned-documentation}
+## वर्ज़न्ड दस्तावेज़ {#versioned-documentation}
 
-This documentation always reflects the latest stable version of React. Since React 16, you can find older versions of the documentation [on a separate page](/versions). Note that documentation for past versions is snapshotted at the time of the release, and isn't being continuously updated.
+ये दस्तावेज़ हमेशा React के लेटेस्ट स्टेबल वर्ज़न पे आधारित होते हैं | आप React 16 के बाद के सारे दस्तावेज़ एक [अलग पेज पर](/versions) पा सकते हैं | ध्यान दें की सारे पिछले वर्ज़न अपने रिलीज़ के समय जैसे के तैसे हैं और उनमे लगातार बदलाव नहीं लाया जाता है |
 
-## Something Missing? {#something-missing}
+## कुछ छूटा है? {#something-missing}
```

Comment:
`कुछ छूटा है?` => `कुछ छूट रहा है?`

Reply:
maybe,
> क्या कुछ छूट रहा है?
 is even better btw I will be doing correction as suggested by you.

Reply:
> क्या कुछ छूट रहा है?

Actually this sounds better, let's update this